### PR TITLE
users: default random avatars

### PR DESCRIPTION
**Pull request by noeldelgado**
- https://monosnap.com/file/z0p4ZNWp7eGbNeLS3nYUsjQNzVoW4Z

### CODE REVIEW

**Comment by asood123** [comment]
It would be nice to be consistent on the avatar that we show for each user. Instead of using random, could we mod on the email address, so it'll always produce the same avatar for a user?

**Comment by piamancini** [comment]
I like that idea 

P.
piamancini.com
w:+54911 5317 4620 
t: @piamancini 
s: piamancini

**Comment by noeldelgado** [comment]
@asood123 ok, I just update it. 
I am using the `user.id` for now because the `email` isn't present on the `user` object right now.

File: frontend/src/components/UserPhoto.js [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-    url: '/static/images/default_avatar.svg',
     addBadge: false
   };
 
   render() {
-    const { className = '', url, addBadge } = this.props;
-    const avatar = (url || '/static/images/default_avatar.svg');
-
+    const { className = '', user, addBadge } = this.props;
+    const userId = (user.id || 0).toString();
```
Comment (asood123) [inline frontend/src/components/UserPhoto.js]: Is toString necessary? We could keep it an Int and simplify the library by modding id with 4.

Comment (noeldelgado) [inline frontend/src/components/UserPhoto.js]: @asood123 I just simplified the function as you requested. :+1:

**Pull request merged**